File: project_euler/00/009.py
#!/usr/bin/env python3
'''Copyright [2017] <dmnsn7@gmail.com>'''

SUM = 1000


def main():
    for i in range(1, SUM + 1):
        for j in range(i, SUM - i + 1):
            if SUM - i - j < j:
                break
            k = SUM - i - j
            if i * i + j * j == k * k:
                multi = i * j * k
                print(multi)


if __name__ == '__main__':
    main()

File: project_euler/00/009.go
// Copyright [2017] <dmnsn7@gmail.com>

package main

import "fmt"

const sum = 1000

func main() {
	for i := 1; i <= sum; i++ {
		for j := i; i+j <= sum && sum-i-j >= j; j++ {
			k := sum - i - j
			if i*i+j*j == k*k {
				multi := i * j * k
				fmt.Printf("%d\n", multi)
			}
		}
	}
}
